lib/pluginmanager/templates/filter-plugin/logstash-filter-example.gemspec.erb

```diff
@@ -0,0 +1,23 @@
+Gem::Specification.new do |s|
+  s.name          = 'logstash-filter-<%= @plugin_name %>'
+  s.version       = '1.0.0'
```

Comment:
I think we should use the default version that the `bundle gem` command is using which is `0.1.0`